src/aliases/api/handlers/handlers.go

```diff
@@ -295,3 +306,14 @@ func validatePathVars(pathVars ...string) error {
 	}
 	return nil
 }
+
+func checkAliasValue(value types.AliasValue) error {
+	for _, v := range value {
+		// we just want the decoding to happen without error
+		_, err := base64.StdEncoding.DecodeString(v)
+		if err != nil {
+			return errors.InvalidFormatError("alias value is incorrect: %v: %v", err, v)
```

Comment:
this function should either return `InvalidFormatError` and then the parent should just return it or this function should return a normal error and the parent wraps it in an `InvalidFormatError`. Currently there is a double return of `InvalidFormatError` from parents and this function.

Reply:
good catch